protobufs/livekit_analytics.proto

```diff
@@ -110,8 +111,18 @@ enum AnalyticsEventType {
   INGRESS_STARTED = 16;
   INGRESS_ENDED = 17;
   INGRESS_UPDATED = 29;
-
-  // NEXT_ID: 30
+  SIP_INBOUND_TRUNK_CREATED = 30;
+  SIP_INBOUND_TRUNK_DELETED = 31;
+  SIP_OUTBOUND_TRUNK_CREATED = 32;
+  SIP_OUTBOUND_TRUNK_DELETED = 33;
+  SIP_DISPATCH_RULE_CREATED = 34;
+  SIP_DISPATCH_RULE_DELETED = 35;
+  SIP_PARTICIPANT_CREATED = 36;
+  SIP_CALL_ACCEPTED = 37;
+  SIP_CALL_STARTED = 38;
+  SIP_CALL_ENDED = 39;
```

Comment:
does this differentiate between inbound and outbound calls? should we have a separate status for `CONNECTING` or is that what STARTED is? for example the flow for inbound:
* CONNECTING -> when sip picks up, but no user on the other side
* ACCEPTED -> when sip connects to end user.

and for outbound
* CONNECTING -> when we start dialing for outbound
* ACCEPTED -> when the other side picks up

Reply:
In the current proposal, PARTICIPANT_CREATED and CALL_ACCEPTED match the "CONNECTING" state you suggest in the outbound and inbound cases respectively. I guess ACCEPTED may be confusing as it means that the SIP server accepted it, not the room. Maybe use "CALL_INCOMING"?

SIP_PARTICIPANT_CREATED has the pro that it relates directly to an API call IMHO.

Reply:
Or CALL_ACCEPTED -> RINGING?

Reply:
got it. so accepted is for incoming only.. makes sense.. then I would vote for `INCOMING`, since it's a bit more explicit about that fact. wdyt?